README.md

```diff
@@ -177,6 +177,15 @@ const Component = () => {
 };
 ```
 
+## Usage with immer
+
+Disabling auto freeze is recommended. JavaScript does not support nested proxies of frozen objects.
+
+```js
+import { setAutoFreeze } from 'immer';
+setAutoFreeze(false);
+```
+
 *   [CodeSandbox](https://codesandbox.io/s/proxy-memoize-demo-yo00p)
```

Comment:
I think we should move this line above the new section.

Reply:
OK, moved the new line up, if that's what you were looking for.

Reply:
I mean this codesandbox link is for zustand usage.

Reply:
Ah, thanks, sure fixed.